src/components/navigation/navigation.spec.js

```diff
@@ -86,6 +86,66 @@ const EXAMPLE_NAVIGATION_WITH_SUBNAVIGATION = {
   },
 };
 
+const EXAMPLE_NAVIGATION_WITH_HIDDEN_SUBNAVIGATION = {
+  navigation: {
+    id: 'main-nav',
+    ariaLabel: 'Main menu',
+    currentPath: '#1',
+    currentPageTitle: 'Main nav item 2',
+    itemsList: [
+      {
+        title: 'Main nav item 1',
+        url: '#0',
+        classes: 'custom-class-main-item-1',
+        id: 'main-item-1',
+      },
+      {
+        title: 'Main nav item 2',
+        url: '#1',
+        classes: 'custom-class-main-item-2',
+        id: 'main-item-2',
+      },
+    ],
+    subNavigation: {
+      id: 'sub-nav-hidden',
+      overviewURL: '#overview',
+      overviewText: 'Overview',
+      ariaLabel: 'Section menu',
+      currentPath: '#1',
+      hideDisplay: true,
+      itemsList: [
+        {
+          title: 'Sub nav item 1',
+          url: '#0',
+          classes: 'custom-class-sub-item-1',
+          id: 'sub-item-1',
+        },
+        {
+          title: 'Sub nav item 2',
+          url: '#1',
+          classes: 'custom-class-sub-item-2',
+          id: 'sub-item-2',
+          sections: [
+            {
+              sectionTitle: 'Section 1',
+              children: [
+                {
+                  title: 'Child item 1',
+                  url: '#0',
+                },
+                {
+                  title: 'Child item 2',
+                  url: '#1',
+                },
+              ],
+            },
+          ],
+        },
+      ],
+    },
+  },
+};
+
```

Comment:
We can probably simplify this config a bit more considering we are only testing for its existence in the html